packages/react-aria-components/src/Table.tsx

```diff
@@ -373,13 +373,16 @@ function Table(props: TableProps, ref: ForwardedRef<HTMLTableElement>) {
     }
   });
 
+  let isListDraggable = !dragAndDropHooks?.isDisabled && hasDragHooks;
+  let isListDroppable = !dragAndDropHooks?.isDisabled && hasDropHooks;
+
   let {selectionBehavior, selectionMode, disallowEmptySelection} = state.selectionManager;
   let ctx = useMemo(() => ({
     selectionBehavior: selectionMode === 'none' ? null : selectionBehavior,
     selectionMode,
     disallowEmptySelection,
-    allowsDragging: isListDraggable
-  }), [selectionBehavior, selectionMode, disallowEmptySelection, isListDraggable]);
+    allowsDragging: hasDragHooks
```

Comment:
Perhaps `allowsDragging` strange name now that `hasDragHooks` doesn't necessarily mean that the table allows dragging depending if `isDisabled` is true but I think we are stuck with the name or it would be a breaking change. Might be useful to have another value on the context here called `isDragDisabled` perhaps?

Commenting so others on the team can weigh in, no need to implement in this PR as of now until we decide on the api here.

Reply:
I think isDragDisabled would be useful. It might come up when we implement edit mode as well.

Reply:
Yeah I imagine it would be common api across several components. We can tackle that later as a followup IMO

Reply:
+1 for having a way of knowing whether the drag&drop is disabled. I am currently building a table that has drag&drop disabled in certain situations and I would really like to only render the drag button when enabled, or at least properly disable the drag button. Is there some other way to achieve this?

Reply:
Mind sharing your setup (perhaps in a github discussion for greater visibility)? I think ideally we'd add `isDragDisabled` or something along those lines to the table options context, but for the time being you can probably just create your own context wrapper around your table implementation that informs your Row if dragging is disabled